app/modules/feed-items/model.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+/* eslint-disable import/prefer-default-export */
+
+import type { Identifier } from 'types/model';
+
+export type FeedItem = {
+  +id: Identifier,
+  +userId: Identifier,
+  +topicId: Identifier,
+  +predicate: string,
```

Comment:
If I recall correctly predicate is supposed to have a limited list of options? If so, try defining these as constants (similar to contentItemTypes, for example) to reduce the risk of typos.

Reply:
not relevant for this PR. Is being worked on in a separate branch.